### ISSUE TITLE
[Android]The library cannot be built on Gradle 8

### ISSUE THREAD

**Issue by theoduong05** [Rapsssito/react-native-background-actions#179]
I recently upgraded from react-native version 0.68.2 to 0.72.0-rc3 and am experiencing issues with the react-native-background-actions library. Has anyone else experienced this issue and can provide guidance on how to fix it? Thank you in advance for your help.
<img width="1424" alt="Screenshot 2023-05-15 at 19 52 21" src="https://github.com/Rapsssito/react-native-background-actions/assets/83207011/e1d75dac-e4c6-4a22-9f7a-beea7b41c05e">

**Comment by AndriyZahura** [comment]
For anyone wondering, for me it happened after I upgraded to Gradle 8.x from 7.x
The solution was found [here](https://discuss.gradle.org/t/namespace-not-specified-for-agp-8-0-0/45850/5)
TL;DR:
Put this in your general build.gradle file
```
subprojects {
   afterEvaluate { project ->
       if (project.hasProperty('android')) {
           project.android {
               if (namespace == null) {
                   namespace project.group
               }
           }
       }
   }
}
```

**Comment by Rapsssito** [comment]
Should be fixed by #208.
